### PR TITLE
Split pages

### PR DESCRIPTION
**Pull request by henryiii**
This is the new, split page homepage with new package list design. Adds an about and packages page. The homepage has more pop and zing.

### CODE REVIEW

**Comment by henryiii** [comment]
Here's what the new homepage looks like:

Screen Shot 2020-03-17 at 4 29 18 PM

**Comment by henryiii** [comment]
And here's what the new projects page looks like:

> Note: it does not add AmpGen, that's just a local file I have, just in case it is ever added, and it shows up when I build locally.

Screen Shot 2020-03-17 at 4 29 59 PM

**Comment by chrisburr** [comment]
That looks really nice though I think this gives a little bit too much prominence to affiliated packages as the green stands out too much. How about swapping the green and grey?

**Comment by henryiii** [comment]
The green is a bit brighter than I intended, too. I could also do blue. Currently the grey matches the sidebar.

**Comment by mayou36** [comment]
Agree with chris, a lighter green or a light blue would maybe match closer to the grey.
But looks good!

**Comment by henryiii** [comment]
Here's #e4f3ff, still applied to affiliated:

Screen Shot 2020-03-17 at 4 50 40 PM

**Comment by henryiii** [comment]
@mayou36 Could I get a higher resolution logo with transparency for zfit?

Also, I need to tweak the behavior of the page for people with 4 inch phone screens (me).

**Comment by jpivarski** [comment]
Is this the darker (non-blue) gray? Most of us on Gitter (myself included) liked that one.

**Comment by henryiii** [comment]
@jpivarski Yes, 50% grey. Removed the dual svg's too, since we had one that had a fixed ring.

logo

**Comment by henryiii** [comment]
Here's what it looks like now with updated menus and the new logos:

Screen Shot 2020-03-18 at 11 19 43 AM

**Comment by henryiii** [comment]
And here's what it looks like on screen as small as my phone's:

Screen Shot 2020-03-19 at 10 20 06 AM

File: pages/about/getting-in-touch.md [hunk 1]
```diff
@@ -2,12 +2,25 @@
 layout: page
 title: Getting in touch
 permalink: /getting-in-touch
-nav_order: 2
+nav_order: 10
```
Comment (eduardo-rodrigues) [inline pages/about/getting-in-touch.md]: Page getting in touch now at the very bottom? You feel that's not better to have it higher up? No strong feeling.

File: pages/about/getting-in-touch.md [hunk 2]
```diff
@@ -2,12 +2,25 @@
...
+parent: About
 ---
 
 Getting in touch {#get_in_touch}
 ================
 
+Gitter
+------
+
+The fastest way to contact us is via Gitter channel
```
Comment (eduardo-rodrigues) [inline pages/about/getting-in-touch.md]:
```suggestion
The fastest way to contact us is via the Gitter channel
```

File: pages/about/getting-in-touch.md [hunk 3]
```diff
@@ -2,12 +2,25 @@
...
+[Scikit-HEP/community][], ideal for
+informal exchanges.
+See also related rooms [HSF/PyHEP][] and [HSF/PyHEP-newcomers][].
```
Comment (eduardo-rodrigues) [inline pages/about/getting-in-touch.md]:
```suggestion
See also related rooms [HSF/PyHEP][] and [HSF/PyHEP-newcomers][] from the HEP Software Foundation (HSF).
```

File: pages/projects/affiliated.md [hunk 4]
```diff
@@ -2,7 +2,8 @@
 layout: page
 title: Affiliated packages
 permalink: /affiliated
-nav_order: 5
+nav_order: 1
```
Comment (eduardo-rodrigues) [inline pages/projects/affiliated.md]: Are the affiliated projects coming prior to the project packages themselves? I may be misinterpreting the 1 here.

**Comment by eduardo-rodrigues** [comment]
And where is the "miscellaneous resources" page? I no longer see it in your picture above.

File: pages/projects/affiliated.md (hunk 4 above)
Comment (henryiii) [reply pages/projects/affiliated.md]: These are now nested. See the screenshot above (not the phone one).

**Comment by henryiii** [comment]
> And where is the "miscellaneous resources" page and a couple others? I no longer see these in your picture above.

They show up when you click about, both as a ToC in the page, and on the left.

File: pages/about/getting-in-touch.md (hunk 1 above)
Comment (henryiii) [reply pages/about/getting-in-touch.md]: I'll have to check the actual order, I'll post a screenshot in a bit - I'm spreading the numbers a bit so it's easier to inject a page without renumbering everything.

Comment (eduardo-rodrigues) [reply pages/about/getting-in-touch.md]: OK, fine.

File: pages/projects/affiliated.md (hunk 4 above)
Comment (eduardo-rodrigues) [reply pages/projects/affiliated.md]: I see.

**Comment by henryiii** [comment]
Screen Shot 2020-03-19 at 11 34 06 AM

It does look like Getting in Touch is at the bottom, I thought this looked reasonable but we can move it.

**Comment by henryiii** [comment]
I'll rename Affiliated Packages to Affiliated projects; the page name for Projects is `/projects.html` so probably would be odd if that changed to be called "Packages" - the title for `/affiliated.html` is already "Affiliated projects and packages".

**Comment by henryiii** [comment]
We can drop the color for the affiliated packages, and just rely on the affiliated note on the right. Would that work?

**Comment by eduardo-rodrigues** [comment]
OK, this snapshot is clear. We should see what other think but my personal opinion is the following:
- "Getting in touch" is sufficient important to be on the main menu rather than semi-hidden in the "About".
- Same for "miscellaneous resources", which, if moved to a sub-menu, is kind of complementary to the documentation.
- The "FAQ" and "Funding" are nicely in the new "About".

**Comment by henryiii** [comment]
I'm happy to make those changes. Keep in mind, the more we put at top level, the less impact the existing ones have.

This will be very useful for the Dev information, I think.

**Comment by henryiii** [comment]
Oh, I added projects, I can rename it to packages, sorry. Thought it was an existing page.

**Comment by henryiii** [comment]
Screen Shot 2020-03-19 at 12 06 30 PM

**Comment by eduardo-rodrigues** [comment]
> We can drop the color for the affiliated packages, and just rely on the affiliated note on the right. Would that work?

I think so.

**Comment by eduardo-rodrigues** [comment]
The developer info link could go to a sub-menu such as the "About" or rather "Resources"?

**Comment by eduardo-rodrigues** [comment]
Pinging @marinang since he prepared in the meantime a logo for `hepstats` :-).

**Comment by henryiii** [comment]
> The developer info link could go to a sub-menu such as the "About" or rather "Resources"?

There will be lots of items in developer eventually. I can see if you can nest in a sub menu.

**Comment by marinang** [comment]
@eduardo-rodrigues unfortunately I don’t have a logo with transparency. I did a quick logo with keynote, and I can’t find the original file... sorry.

**Comment by eduardo-rodrigues** [comment]
> @eduardo-rodrigues unfortunately I don’t have a logo with transparency. I did a quick logo with keynote, and I can’t find the original file... sorry.

Ah,  shame. But it probably does not quite matter for the website.

**Comment by henryiii** [comment]
Pages do not nest properly multiple levels. When you go to an inner level, the navigation resets to the top level. I think developer information is going to be quite useful as well once we have more guides there.

**Comment by henryiii** [comment]
We can set this to be always expanded with:

```css
.navigation-list .navigation-list-item .navigation-list-child-list {
  display: block;
}
```

And the color can be changed with:

```css
.navigation-list .navigation-list-child-list .navigation-list-link {
  color: #7092be;
}
```

At the moment, I think it looks pretty good, though. Thoughts?

**Comment by eduardo-rodrigues** [comment]
Yes, I think so. We can always fine-tune any loose ends once we see all new pages online.
Thanks!

**Comment by henryiii** [comment]
And here's what saving it in your favorites looks like now! (hint, hint)

image

**Pull request merged**